portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,7 +53,12 @@
   private static final String NAME = "name";
   private static final String COMMENT_TEXT = "commentText";
   private static final String EMAIL = "email";
-  
+
+  // Supported image files.
+  private static final String JPEG = "image/jpeg";
```

Comment:
Optional: consider creating: List<String> supportedImageFormats 

This way you do not need to create 1 variable per image format.